README.md

```diff
@@ -27,14 +27,14 @@ For Maven add to your `pom.xml`:
   <dependency>
     <groupId>io.opencensus</groupId>
     <artifactId>opencensus-api</artifactId>
-    <version>0.10.1</version>
+    <version>0.11.0</version> <!-- LATEST_OPENCENSUS_RELEASE_VERSION -->
```

Comment:
Let me try that. Another option is to use a more complex regex. I think the motivation for adding comment is just to avoid substituting other components' version that happen to be the same as ours.